docs/git.md

```diff
@@ -0,0 +1,74 @@
+# Git Workflow
+
+## Table Of Contents
+
+[Git Commit Guidelines](#commit)
+  - [Commit Message Format](#format)
+  - [Type](#type)
+  - [Subject](#subject)
+  - [Emoji](#emoji)
+  - [Body](#body)
+
+
+## <a name="commit"></a> Git Commit Guidelines
+
+We have very precise rules over how our git commit messages can be formatted. This leads to **more
+readable messages** that are easy to follow when looking through the **project history**.  But also,
+we use the git commit messages to **create change log**.
```

Comment:
I thought I had convinced you ;-)

More seriously, this wish is a *work-in-progress* reflexion, so it may not be written black on white on the guideline.